consensus/istanbul/validator/multi_staking_test.go

```diff
@@ -192,6 +195,7 @@ func TestCalcWeight(t *testing.T) {
 			[]float64{0, 0, 0},
 			0,
 			[]uint64{0, 0, 0},
+			params.Rules{ChainID: big.NewInt(0), IsIstanbul: false, IsLondon: false, IsMagma: false, IsKore: false},
```

Comment:
It's a minor thing, but can we use global variables instead?
```go
var (
    magmaRules = params.Rules{ChainID: common.Big0, IsIstanbul: true, IsLondon: true, IsMagma: true, IsKore: false}
    koreRules = params.Rules{ChainID: common.Big0, IsIstanbul: true, IsLondon: true, IsMagma: true, IsKore: true}
)
```

Reply:
Refleted in `175fa32081957ff10630c2106f930d6ad5194cbc`.
Thank you!